README.md

```diff
@@ -59,6 +60,14 @@ $ brew tap sgoudham/tap
 $ brew install uwuifyy
 ```
 
+### AUR
```

Comment:
Thanks for going ahead and packaging for arch users!

Just one comment here, a recent PR (#11) was merged that contains and uwu'ified version of the README.md. I believe `README.uwu.md` should also be updated as part of this change! Of course, with the text uwu'ified.

In the future, I can maybe look towards automating this process but usually some manual tweaks are needed.